sig-auth/cluster-role-aggregation.md

```diff
@@ -0,0 +1,94 @@
+# Cluster Role Aggregation
+In order to support easy RBAC integration for CustomResources and Extension
+APIServers, we need to have a way for API extenders to add permissions to the
+"normal" roles for admin, edit, and view.
```

Comment:
This isn't explicitly stated, but it seems to be implied that we see admin, edit, and view being switched to aggregated rules so admins can register their permissions on the default ClusterRoles?

Reply:
> This isn't explicitly stated, but it seems to be implied that we see admin, edit, and view being switched to aggregated rules so admins can register their permissions on the default ClusterRoles?

Yes.  The open pull demonstrates a simple way to transition them.